docs/content/configuration.md

```diff
@@ -393,6 +393,7 @@ included in the actual bundle gzipped tarball.
 | `bundles[_].signing.keyid` | `string` | No | Name of the key to use for bundle signature verification. |
 | `bundles[_].signing.scope` | `string` | No | Scope to use for bundle signature verification. |
 | `bundles[_].signing.exclude_files` | `array` | No | Files in the bundle to exclude during verification. |
+| `bundles[_].size_limit_bytes` | `int64` | No (default: `1073741824`) | Size limit for bundle files. |
```

Comment:
nit: We may want to reword this a tiny bit, IMO there is some ambiguity whether this means its the size limit for a bundle file (eg `bundle.tar.gz`) or for a single uncompressed file in the bundle (which is what it actually limits).